frontend/src/components/MeetingItem/MeetingItem.styled.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+
+export const Layout = styled.li`
+  min-width: 160px;
```

Comment:
width로 바꾸니 스타일이 적용되네요.

```suggestion
  width: 160px;
```

Reply:
rem으로 수정했습니다~